lib/v0_bindings_extensions.rb

```diff
@@ -123,11 +133,21 @@ def update_model!(model)
 
   Api::V0::Bindings::GetHighlightsSummaryParameters.class_exec do
     def summarize(user_uuid:)
-      highlights = ::Highlight.by_user(user_uuid)
+      users = []
+
+      if sets.blank? or sets.include?('user:me')
+        users.push(user_uuid)
+      end
+      if  sets.to_a.include?('curated:openstax') and scope_id.present?
+        curator_scope = CuratorScope.find_by(:scope_id => scope_id)
+        users.push(curator_scope.curator_id) unless curator_scope.nil?
+      end
```

Comment:
To DRY this up, recommend a `sets_to_user_ids` method somewhere or similar.

Reply:
i wasn't sure where to put this, for now i put it in the shared monkey patch, but the whole thing makes me feel dirty.